ceph.spec.in

```diff
@@ -846,8 +846,8 @@ mkdir -p %{_localstatedir}/run/ceph/
 %{_datadir}/ceph/id_dsa_drop.ceph.com
 %{_datadir}/ceph/id_dsa_drop.ceph.com.pub
 %dir %{_sysconfdir}/ceph/
-%dir %{_localstatedir}/log/ceph/
-%dir %{_datarootdir}/ceph/
+%attr(3770,ceph,ceph) %dir %{_localstatedir}/log/ceph/
```

Comment:
This is mostly unnecessary, the permission for /var/log/ceph are specified in the filelist but later. (~line 855; they still apply, though)

We should probably remove this line altogether. You might want to change the other line to have the 3770 perms though. Currently, it says 2750.